src/game/GameManager.h

```diff
@@ -135,6 +137,10 @@ class GameManager {
     GameHashMap<int32_t, std::shared_ptr<Store>> storeManager;
     GameHashMap<int32_t, DropPoint> dropPointManager;
     std::vector<int32_t> openDropPoints;
+
+    int dropPointSpace;
+    int storeSize;
+    int storePickupSize;
```

Comment:
these should never be member variables
follow the implementation guide on how to create constants